helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -357,6 +355,38 @@ public void scheduleRebalance(long rebalanceTime) {
     }
   }
 
+  /**
+   * schedule an on demand rebalance pipeline.
```

Comment:
If you do a /** and hit enter, the IDE creates for you a pretty block comment. Could you try that?